Firestore/core/src/remote/serializer.cc

```diff
@@ -1429,9 +1430,30 @@ std::unique_ptr<WatchChange> Serializer::DecodeDocumentRemove(
 
 std::unique_ptr<WatchChange> Serializer::DecodeExistenceFilterWatchChange(
     ReadContext*, const google_firestore_v1_ExistenceFilter& filter) const {
-  ExistenceFilter existence_filter{filter.count};
-  return absl::make_unique<ExistenceFilterWatchChange>(existence_filter,
-                                                       filter.target_id);
+  ExistenceFilter existence_filter = DecodeExistenceFilter(filter);
+  return absl::make_unique<ExistenceFilterWatchChange>(
+      std::move(existence_filter), filter.target_id);
```

Comment:
Consider replacing `std::move(existence_filter)` with just `DecodeExistenceFilter(filter)`, avoiding the `existence_filter` local variable.